README.md

```diff
@@ -75,3 +75,4 @@ Table of Content
 ## Other Resources
 
 - [Tezos Stackexchange](https://tezos.stackexchange.com/questions)
+- [Tezos URI standard (payment requests)](https://gitlab.com/tips2/TIPs/blob/master/TIPS/tip-8.md) and a [library providing an easy way to add to apps and wallets](https://gitlab.com/smartcontractlabs/tezos-uri)
```

Comment:
I'm not sure we should include TIPS in the draft form.
I'm also not really aware how TIPS fits into the amendment protocol (the digestible introduction I used for understanding how amendments get proposed + added to Tezos was written by nomadic labs [here](https://blog.nomadic-labs.com/amendments-at-work-in-tezos.html))

Do you mind elaborating on TIPS a bit (how does it work>, who in the community uses it,? etc)